src/views/IdentificationConfig.vue

```diff
@@ -0,0 +1,319 @@
+                    'Standard-Fahrzeug zurückgesetzt. Die Option ist im Lade-Profil "' +
+                    $store.state.mqtt[
+                      "openWB/vehicle/template/charge_template/" +
+                        $store.state.mqtt["openWB/vehicle/" + vehicleId + "/charge_template"]
+                    ].name +
+                    '" gespeichert.'
+                  }}
+                  <div
+                    v-if="
+                      $store.state.mqtt[
+                        'openWB/vehicle/template/charge_template/' +
+                          $store.state.mqtt['openWB/vehicle/' + vehicleId + '/charge_template']
+                      ].load_default === true
+                    "
+                  >
+                    "Standard-Fahrzeug nach Abstecken" ist für alle Fahrzeuge aktiviert, denen dieses Lade-Profil
+                    zugeordnet wurde.
+                  </div>
+                  <div v-else>
+                    "Standard-Fahrzeug nach Abstecken" ist für alle Fahrzeuge deaktiviert, denen dieses Lade-Profil
+                    zugeordnet wurde.
+                  </div>
+                </template>
+              </openwb-base-button-group-input>
+            </div>
+          </div>
+        </openwb-base-card>
+      </div>
+      <openwb-base-submit-buttons
+        form-name="optionalComponentsForm"
+        @save="$emit('save')"
+        @reset="$emit('reset')"
+        @defaults="$emit('defaults')"
+      />
+    </form>
+  </div>
+</template>
+
+<script>
+import ComponentState from "../components/mixins/ComponentState.vue";
+
+export default {
+  name: "IdentificationConfigView",
+  mixins: [ComponentState],
+  emits: ["save", "reset", "defaults"],
+  data() {
+    return {
+      mqttTopicsToSubscribe: [
+        "openWB/general/extern",
+        "openWB/chargepoint/+/config",
+        "openWB/chargepoint/+/get/rfid",
+        "openWB/chargepoint/+/get/rfid_timestamp",
+        "openWB/chargepoint/+/set/rfid",
+        "openWB/optional/rfid/active",
+        "openWB/chargepoint/+/config",
+        "openWB/chargepoint/template/+",
+        "openWB/vehicle/template/charge_template/+",
+        "openWB/vehicle/+/name",
+        "openWB/vehicle/+/charge_template",
+        "openWB/vehicle/+/tag_id",
+      ],
+      tempIdTagList: {},
+    };
+  },
+  computed: {
+    idTagList() {
+      return Object.values(this.updateIdTagList());
+    },
+    chargePointTemplates: {
+      get() {
+        return this.getWildcardTopics("openWB/chargepoint/template/+");
+      },
+    },
+    vehicleIndexes: {
+      get() {
+        return this.getWildcardIndexList("openWB/vehicle/+/name");
+      },
+    },
+  },
+  methods: {
+    getIdFromTopic(topic) {
+      return topic.match(/(?:\/)([0-9]+)(?=\/)*/g)[0].replace(/[^0-9]+/g, "");
+    },
+    updateIdTagList() {
+      Object.entries(
+        // get all id-tag topics/values
+        this.getWildcardTopics("^openWB/chargepoint/[^+/]+/[gs]et/rfid$", true),
```

Comment:
So werden auch die Topics von der Pro mit angezeigt.
```suggestion
        this.getWildcardTopics("^openWB/chargepoint/[^+/]+/[gs]et/(rfid|vehicle_id)$", true),
```